src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -114,4 +146,42 @@ public void loadFile(File file) {
         // CONTINUE!!!
         System.out.println("loaded");
     }
+
+    //Function to show a description of the function of this app
+    private void showFunctions(){
+        JOptionPane.showMessageDialog(null, "<html>Username, Password, and Sign In:" +
```

Comment:
We can try to use Java 17's feature here, instead of concatenating a whole bunch of strings together, you can do something like what I did: 

```
String query = """
                    CREATE TABLE IF NOT EXISTS users
                    (
                        user_id  INT AUTO_INCREMENT
                                 PRIMARY KEY,
                        username VARCHAR(10) NOT NULL,
                        password VARCHAR(16) NOT NULL,
                        bill_id  INT         NOT NULL
                    );
                    """;
```

Surrounding code by 3 double quotes makes it a super long string

Reply:
Also, using this way, we can also get rid of all the <br>'s, as Java automatically uses \n in the Super Long Strings.